Lib/idlelib/autocomplete_w.py

```diff
@@ -241,8 +243,10 @@ def winconfig_event(self, event):
         acw.wm_geometry("+%d+%d" % (new_x, new_y))
 
     def hide_event(self, event):
+        # This will be trigger when focus on widget or autocompletewindow
         if self.is_active():
-            self.hide_window()
+            if self.widget == self.widget.focus_get() or not self.widget.focus_get():
```

Comment:
2nd if is equivalent to 'and', and line is too long.  This works and conforms to PEP8.
        if (self.is_active() and
                (self.widget == self.widget.focus_get()
                or not self.widget.focus_get())):
            self.hide_window()